src/App.js

```diff
@@ -107,10 +106,11 @@ export default function App() {
                 path="/us/:stateId/county/:countyId/compare/:sharedComponentId?"
                 component={LocationPage}
               />
-              <Route exact path="/learn" component={Landing} />
+              <Route exact path="/explained" component={Landing} />
```

Comment:
We need to add a redirect for `/learn` to ensure that links in the wild are not broken